parm/fv3lam_rrfs.xml

```diff
@@ -385,55 +385,22 @@
        <scale>4.0</scale>
        </param>
 
-       <param>
-       <shortname>TSOIL_ON_DEPTH_BEL_LAND_SFC_3m</shortname>
-       <pname>TSOIL</pname>
-       <level>3.</level>
-       <scale>3.0</scale>
-       </param>
-
        <param>
        <shortname>TSOIL_ON_DEPTH_BEL_LAND_SFC</shortname>
        <pname>TSOIL</pname>
-       <scale_fact_fixed_sfc1>2 2 2 2</scale_fact_fixed_sfc1>
-       <level>0. 10. 40. 100.</level>
-       <scale_fact_fixed_sfc2>2 2 2 2</scale_fact_fixed_sfc2>
-       <level2>10. 40. 100. 200.</level2>
+       <level>0. 1. 4. 10. 30. 60. 100. 160. 300.</level>
+       <level2>0. 1. 4. 10. 30. 60. 100. 160. 300.</level2>
```

Comment:
One question: do fv3lam_rrfs.xml and postxconfig-NT-fv3lam_rrfs.txt work for NOAA MP LSM?  I know EMC also runs a version of RRFS at WCOSS but they use NOAA MP LSM.

Reply:
I believe *fv3lam_rrfs* files are only for our GSL runs.  Maybe @hu5970 can confirm that...